activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-jpa/src/main/java/org/activiti/cloud/services/modeling/jpa/ModelRepositoryImpl.java

```diff
@@ -96,19 +96,16 @@ public ModelRepositoryImpl(ModelJpaRepository modelJpaRepository){
 
     @Override
     public ModelEntity createModel(ModelEntity model) {
-
```

Comment:
If you are not changing the file, but just formatting it, please, can avoid to commit it?